src/components/authentication/login/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Form, Input, Button, Checkbox } from "antd";
+import { Link } from "react-router-dom";
+import { errorMessages } from "../../../constants/errorMessages";
+import { inputRegexes } from "../../../constants/inputRegexes";
+import { login } from "../../../services/authentication";
+import styles from "../styles.module.css";
+
+function Login() {
+    let history = useHistory();
+
+    const onFinish = (values) => {
+        login(values, history);
+    };
+
+    function onChange(e) {
+        console.log(`remember me = ${e.target.checked}`);
```

Comment:
A function that in fact does nothing except displaying a message in the console when the status of the check box changes. What is it for?

Reply:
@VadimChorrny look through the code and remove all console.log's.